tests/coverage/coverage.cpp

```diff
@@ -33,9 +33,7 @@ TEST(Coverage, connectionComment)
         "  </component>\n"
         "  <connection component_1=\"component1\">\n"
         "    <!-- Comment on connection. -->\n"
-        "    <map_variables variable_1=\"variable1\" variable_2=\"variable2\">"
-            "<!-- Comment in a map variables. -->"
-            "</map_variables>"
+        "    <map_variables variable_1=\"variable1\" variable_2=\"variable2\"><!-- Comment in a map variables. --></map_variables>\n"
```

Comment:
Shouldn't we rather have:

```
        "    <map_variables variable_1=\"variable1\" variable_2=\"variable2\">\n"
        "        <!-- Comment in a map variables. -->\n"
        "    </map_variables>\n"
```

So that it's easier to read...?

Reply:
Can't it doesn't have the same meaning.

Reply:
(Obviously) agreed that it doesn't have the same meaning. It's just that it would be better if it could read that way unless the test really doesn't make sense with those `\n` characters.

Reply:
This test is for the purpose of satisfying llvm coverage reporting.  This line is the tidiest I have for triggering the code path.

I'll have another look at the code in question and see if changes there would work better.  This is a very edge case which probably could be re-worked into a better code path.

Reply:
This has been addressed with changes to handling grandchildren of connections.